lib/burndown_data.rb

```diff
@@ -75,6 +76,26 @@ def to_hash
     end
   end
 
+  def to_json
+    { date: date_time.to_s,
+
+      total_sp: story_points.total.round,
```

Comment:
Don't use abbreviations in names. This makes the code or the data structures much harder to read and understand. So better call it `total_story_points`.

Same applies to the other names.